doc/topics/windows/windows-package-manager.rst

```diff
@@ -1052,7 +1027,7 @@ Here's an example using the GUID to uninstall software.
         uninstall_flags: '/qn /norestart'
         msiexec: True
 
-Here's an example using the same MSI used to install the software:
+Here's an example of using the installer MSI to uninstall software
```

Comment:
**Nitpick:** End this sentence with a colon (:) instead.

Reply:
Updated